build.gradle.kts

```diff
@@ -148,6 +148,7 @@ tasks {
   runIde {
     enabled = true
     environment("ENVIRONMENT", "LOCAL")
+    autoReloadPlugins.set(false) // is triggered when building llama server
```

Comment:
I had problems with this default feature while developing, after the llama server is compiled Intellij would sometimes try to reload the CodeGPT plugin because files in the `build/llama.cpp` folder changed, e.g.:
`2024-04-20 10:27:00,055 [  15414]   INFO - #c.i.i.p.DynamicPluginVfsListener - Detected plugin .jar file change /home/user/IdeaProjects/git/CodeGPT/build/idea-sandbox/plugins/CodeGPT/llama.cpp/llama.log, reloading plugin`